run_service.sh

```diff
@@ -1207,5 +1222,17 @@ cd ..
 add_volume_to_service "$PWD/trader_service/abci_build/docker-compose.yaml" "trader_abci_0" "/data" "$path_to_store"
```

Comment:
@0xArdi This only affects Docker Compose deployments. Is there any action required for Kubernetes?

Reply:
Yes there is, we need to update the generated yamls to include the 'data' volume. It's a bit more involved than doing it in docker-compose.